lookout/style/format/langs/__main__.py

```diff
@@ -44,24 +58,25 @@ def extract_node_token(file: str, node: bblfsh.Node) -> str:
     return file[spos.offset:epos.offset].lower()
 
 
-def analyze_uast(path: str, root: bblfsh.Node, roles: set, reserved: set):
-    contents = Path(path).read_text()
-
+def analyze_uast(path: str, content: str, root: bblfsh.Node, roles: dict, uast_roles: dict, reserved: set):
```

Comment:
Line length is 107 here. And hey, how could you pass CI then? Please make the CI fail on long lines!!!

Reply:
no idea :) checking...